src/Amazon.Lambda.Tools/BuildLambdaZip/build-lambda-zip.go

```diff
@@ -27,20 +28,37 @@ func main() {
 			Value: "",
 			Usage: "output file path for the zip. Defaults to the first input file name.",
 		},
+		cli.StringFlag{
+			Name:  "input, i",
+			Value: "",
+			Usage: "input file path for the list of files to zip.",
+		},
 	}
 
 	app.Action = func(c *cli.Context) error {
-		if !c.Args().Present() {
-			return errors.New("No input provided")
-		}
-
 		firstArg := c.Args().First()
```

Comment:
I don't think the `firstArg` value is valid anymore here since the list of files could instead be within the `input` arg's file.

To maintain the same functionality (line 42) the `outputZip` setting should probably be after line 59 and the first entry in the `files` slice.

Reply:
Made this change